vllm/model_executor/models/mixtral.py

```diff
@@ -435,7 +435,7 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                     continue
                 name = name.replace(weight_name, param_name)
                 # Skip loading extra bias for GPTQ models.
-                if name.endswith(".bias") and name not in params_dict:
+                if name.endswith("bias") and name not in params_dict:
```

Comment:
Why do you need to make this change? It is best to keep the period to reduce the potential of false positives

Reply:
It's because GPTQ fused Marlin MoE uses parameters which are named `w13_<parameter_name>` and `w2_<parameter_name>` - so we need to cover both `.bias` and `_bias`. This can be narrowed down to cover only these two cases though.